crates/sdk/Cargo.toml

```diff
@@ -0,0 +1,81 @@
+[package]
+name = "blueprint-sdk"
+version = "0.1.0"
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+
+[features]
```

Comment:
Make sure to use `dep:` syntax here, so all the optional dependencies aren't creating their own features.